### PR TITLE
Redo RDS logic and update Readme

### PR DESCRIPTION
**Pull request by abukharov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ Revolver currently supports the following AWS resources:
  * EC2 instances including those run in Autoscaling, Revolver will pause the ASG prior to shutting down instances
  * EBS volumes. Revolver will tag these resources. Revolver can get the tags from the parent instance/volume.
  * Snapshots. Revolver will tag these resources. Revolver can get the tags from the parent instance/volume.
- * RDS single instances, Revolver will use the native start/stop featur
- * RDS multi-az instances, Revolver will snapshot the instance and delete it. In the morning it will be restored from snapshot
- * RDS Clusters, Revolver will snapshot the cluster and save the critical information about the cluster members on the snapshot
+ * RDS single instances, Revolver will use the native start/stop feature
+ * RDS multi-az instances, Revolver will use the native start/stop feature
+ * RDS Clusters, Revolver will use the native start/stop feature
```
Comment (lyricnz) [inline README.md:32]: Do you need to explicitly say the mechanism, now that all are native?

File: README.md [hunk 2]
```diff
@@ -272,38 +211,35 @@ To validate several tags include this plugin in the configuration once for every
 |match|JS-compatible regular expression to match the value against (optional)|JS regex| - |
 |tag_missing|List of actions to perform on the resource if the tag is missing|`warn`,`stop`| - |
 |tag_not_match|List of actions to perform on the resource if the tag does not match the regex in `match`|`warn`,`stop`| - |
-|allow_set_from_parent|Allow Revolver to try to get tags from parent (instance/volumes) -- works only with ebs/snapshot drivers|`true`,`false`|`true`|
 
 ```
```
Comment (lyricnz) [inline README.md:215]: markdown code block language please

File: README.md (hunk 1 above)
Comment (abukharov) [reply README.md:32]: explicit is better than implicit. also you can track on the diff what the changes are.

File: README.md (hunk 2 above)
Comment (lyricnz) [inline README.md:215]: markdown code block language please

Comment (abukharov) [reply README.md]: done

**Review by lyricnz — APPROVED** [review]
🐒

**Pull request merged**